charts/helm-project-operator/values.yaml

```diff
@@ -177,7 +177,7 @@ debugLevel: 0
 cleanup:
   image:
     repository: rancher/shell
-    tag: v0.1.14
+    tag: v0.1.18-rc4
```

Comment:
will need to come back and make this a non-RC version once rancher/shell is released?

Reply:
I've PR open https://github.com/rancher/shell/pull/33 to update rancher/shell again and make It a official release. As soon as it's merged, I'll update this PR, so we don't need to do it twice.